static/src/js/components/Panels/Panels.jsx

```diff
@@ -561,6 +561,9 @@ export class Panels extends PureComponent {
     panelSectionProps.isOpen = !!(panelSectionProps.panelSectionId === activePanelSectionId)
     panelSectionProps.isActive = !!(panelSectionProps.panelSectionId === activePanelSectionId)
 
+    // If the panelSection is not active, do not render it
```

Comment:
This is so that we can reduce how much render we are doing on this page when its not active?